.gitignore

```diff
@@ -1,2 +1,10 @@
+*.iml
+.gradle
+/local.properties
+/.idea
 .DS_Store
+/build
+/example/build
+/annotation-processor/build
+/captures
```

Comment:
Are these changes needed? Any of these things should be in the android-kotlin/.gitignore specifically.

Reply:
oh, didn't realize I had changed the main .gitignore file, I'll revert